scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/SchedulerServiceImpl.java

```diff
@@ -444,12 +444,11 @@ public InputStream apply(RestClient restClientProxy, Integer jobId) {
     @Override
     public void setPriorityByName(final String sessionId, List<Integer> jobIdList, final String priorityName)
             throws ServiceException, RestServerException {
-        executeFunction(new BiFunction<RestClient, Integer, InputStream>() {
+        executeVoidFunction(new BiFunction<RestClient, Integer, Void>() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)